Login And Signup Page Design

NishitModia opened the pull request:
change "." to "!".

NishitModia committed:
change "." to "!".

NishitModia committed:
changed "!" in conn.

GautamMakadia reviewed (changes requested):
It's not sufficient enough to be available in main branch, Make some appropriate changes which could be significantly improve project.

NishitModia committed:
Add new CSS File and changing in Login.html and Sign up.html.

GautamMakadia reviewed (changes requested):
I have viewed and executed your changes, It seems fine, but you should move .css files to the assets folder, and then update it in html files.
![Screenshot from 2023-07-19 15-33-49](https://github.com/GautamMakadia/Wall_Rex/assets/69979317/39918fd8-fcc9-493d-8dcb-38ac139f487c)
![Screenshot from 2023-07-19 15-33-56](https://github.com/GautamMakadia/Wall_Rex/assets/69979317/4dfc1f5e-d060-44ac-a05a-dc7f477c4f33)

GautamMakadia committed:
Code & Project Structure Refactor
- Moved p2.jpg file to ../image/ folder.
- Moved index.css file to ../assets/login-forms.css
- Updated action & method of forms.
- Removed some unwanted comments from html pages.
